docs/README.md

```diff
@@ -1,90 +1,41 @@
-### Note for external contributors
+# Building the docs locally
 
-We are currently working on migrating the docs to a new static website. The `grafana/website` repository will be private during this migration, which unfortunately means the docs site can't be built without access.
+## Requirements
 
-The Markdown content however, is still public in this repository. We still encourage pull requests to make the docs better, and we will make sure the changed content works well on the current docs site. Include any images in your pull request, and we will move them to the `grafana/website` repository.
+Docker >= 2.1.0.3
 
-# Building the docs
+## Getting Started
 
-To build the docs locally, you need to have Docker installed. The docs are built using [Hugo](http://gohugo.io/) - a static site generator.
+To build the docs locally:
 
-**Prepare the Docker image**:
+1. Make sure you are in the docs folder: `cd docs`.
+2. Run `make build`. This builds the docker image with the correct version of Hugo.
+3. Run `make run`. This runs the container. It will start a hugo sever, and you can naviagate to the site in your browser at `localhost:1313`. 
 
-> Due to migration to new static site, the Docker image needs to be built from `old-docs` branch.
+You can then edit files in the `sources` directory which will update `localhost:1313` when changes are saved.
 
-Git clone `grafana/website` repo. Run these commands in the root of that repo. **Note** that you may require `sudo`
-when running `make docs-build` depending on how your system's Docker
-service is configured):
+---
 
-```
-git clone https://github.com/grafana/website
-cd website
-make docs-build
-```
+## Content Guidelines
 
-**Build the Documentation**:
+Edit content in the `sources` directory.
 
-Now that the Docker image has been prepared we can build the
-grafana docs and start a docs server.
+### Using `relref` for internal links
 
-If you have not cloned the Grafana repository already then:
+Use the hugo shortcode [relref](https://gohugo.io/content-management/cross-references/#use-ref-and-relref) anytime you are linking to other internal docs pages.
 
-```
-cd ..
-git clone https://github.com/grafana/grafana
-```
+### Editing the Sidebar Menu
 
-Switch your working directory to the directory this file
-(README.md) is in.
+Edit [sources/menu.yaml](sources/menu.yaml) to make changes to the sidebar. Restart the `make run` command for changes to take effect.
```

Comment:
Please clarify "Restart the `make run` command." Do you mean run it again? Do we need to stop the watch process first?